teachertool/src/components/styling/shared.scss

```diff
@@ -0,0 +1,261 @@
+/**
+ * This file contains all the shared styles for application components. Do not
+ * import this file directly into components. Instead, import into the
+ * component's module.scss file so that it will be scoped to that module.
+ *
+ * Classes in this file are composable, modeled after [Tailwind CSS](https://tailwindcss.com)
+ */
+
+ // Width
+.w-full {
+    width: 100%;
+}
+.w-screen {
+    width: 100vw;
+}
+.w-auto {
+    width: auto;
+}
+
+// Height
+.h-full {
+    height: 100%;
+}
+.h-screen {
+    height: 100vh;
+}
+.h-auto {
+    height: auto;
+}
+
+/// Visibility
+.hidden {

```

Comment:
Does this already exist in the react-common styles? I think it might.

Reply:
Probably. I ended up not using them anyway. Removed.